network/ipv6.go

```diff
@@ -56,6 +56,14 @@ const (
 	RegionUSEast4    Region = 0x0C
 )
 
+type SuperRegion uint8
+
+const (
+	SuperRegionUSCentral SuperRegion = 0x01
+	SuperRegionUSWest    SuperRegion = 0x02
+	SuperRegionUSEast    SuperRegion = 0x03
+)
```

Comment:
can we make these the short code? 
```suggestion
type SuperRegion string

const (
	SuperRegionUSCentral SuperRegion = "usc"
	SuperRegionUSWest    SuperRegion = "usw"
	SuperRegionUSEast    SuperRegion = "use"
)
```

itll cost us like 30 more bytes but save some paperwork